README.md

```diff
@@ -99,9 +99,8 @@ git submodule update --remote --merge
 This project uses [openapi-python-client](https://github.com/openapi-generators/openapi-python-client) to generate an API client from the IDL. To update the generated client:
 
 ```bash
-mkdir iwf/iwf_api/iwf_api
-cd iwf && poetry run openapi-python-client update --path ../iwf-idl/iwf-sdk.yaml --config .openapi-python-client-config.yaml
-cd .. && cp -R iwf/iwf_api/iwf_api/* iwf/iwf_api && rm -R iwf/iwf_api/iwf_api && poetry update
+poetry run openapi-python-client generate --path iwf-idl/iwf-sdk.yaml --config iwf/.openapi-python-client-config.yaml
+cp -R iwf_api/iwf_api/* iwf/iwf_api && rm -R iwf_api/ && poetry update
```

Comment:
The command that was here before never worked for me. Not sure if I'm the only one with this issue, but it should work for everyone now.